tests/models/test_taskinstance.py

```diff
@@ -1547,6 +1550,47 @@ def test_pendulum_template_dates(self):
         assert isinstance(template_context["next_execution_date"], pendulum.DateTime)
         assert isinstance(template_context["prev_execution_date"], pendulum.DateTime)
 
+    @parameterized.expand(
+        [
+            ('{{ conn.get("a_connection").host }}', 'hostvalue'),
+            ('{{ conn.get("a_connection", "unused_fallback").host }}', 'hostvalue'),
+            ('{{ conn.get("missing_connection", {"host": "fallback_host"}).host }}', 'fallback_host'),
+            ('{{ conn.a_connection.host }}', 'hostvalue'),
+            ('{{ conn.a_connection.login }}', 'loginvalue'),
+            ('{{ conn.a_connection.password }}', 'passwordvalue'),
+            ('{{ conn.a_connection.extra_dejson["extra__asana__workspace"] }}', 'extra1'),
+        ]
+    )
+    def test_template_with_connection(self, content, expected_output):
```

Comment:
this test cover only connection defined in the database.
connection can also be in secret backend - will the macro work with it? shouldn't it be also covered in test?

Reply:
It's covered already by the unit tests of the methods used `get_connection_from_secrets`. There is no need to repeat those.